opengrok-indexer/src/main/java/org/opengrok/indexer/util/StatUtil.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+ /*
+  * Copyright (c) 2006, 2018, Oracle and/or its affiliates. All rights reserved.
```

Comment:
no 2006

Reply:
I think it needs to be preserved, assuming the code comes from at least 2006. I am not a lawyer, though.

Reply:
ok